src/components/chat/ContextTokenIndicator.tsx

```
@@ -117,81 +125,56 @@ const ContextTokenIndicator: React.FC<ContextTokenIndicatorProps> = ({
-    return () => {
-      window.removeEventListener('mousemove', handleMouseMove);
-      if (hideTimer.current) {
-        clearTimeout(hideTimer.current);
+
+      if (!isVisible) {
+        return;
       }
-    };
-  }, [isMobile, showContextTokenIndicator, resetHideTimer, getTriggerArea, isVisible]);
+
+      const cardElement = document.querySelector('[data-testid=\"context-token-card\"]') as HTMLElement | null;
```

ESLint

[error] Unnecessary escape character: \".

(no-useless-escape)